workflow/pipeline/core.py

```diff
@@ -27,15 +27,15 @@
 session.activate(db_prefix + 'session', linking_module=__name__)
 
 
-# Declare SkullReference table for use in element-array-ephys
+# Declare SurgeryCoordinateReference table for use in element-array-ephys
 @lab.schema
-class SkullReference(dj.Lookup):
+class SurgeryCoordinateReference(dj.Lookup):

```

Comment:
Can we name this `BrainCoordinateReference`?

Reply:
ok, I'll make that change.